src/main/java/frc/robot/RobotContainer.java

```diff
@@ -161,8 +181,11 @@ public double getAsDouble() {
   DoubleSupplier m_rotationSupplier = new DoubleSupplier() {
     @Override
     public double getAsDouble() {
+      if (_hangarAlign) {
```

Comment:
Why does hangarAlign take priority over limelight align? I think we should wait to add this until after the refactor